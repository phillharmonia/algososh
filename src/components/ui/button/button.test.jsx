
import { render, screen, fireEvent } from '@testing-library/react';
import TestRenderer from 'react-test-renderer';
import {Button} from "./button";

describe('Тест Button', () => {
    it('Тест кнопки с текстом', () => {
        const button = TestRenderer.create(<Button text='Кнопка' />).toJSON();
        expect(button).toMatchSnapshot()
    })
    it('Тест кнопки без текста', () => {
		const button = TestRenderer
			.create(<Button />)
			.toJSON()
		expect(button).toMatchSnapshot()
    })
})