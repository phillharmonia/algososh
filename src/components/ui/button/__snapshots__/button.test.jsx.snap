// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Тест Button Тест кнопки без текста 1`] = `
<button
  className="text text_type_button text_color_primary button undefined false "
  type="button"
>
  <p
    className="text undefined"
  />
</button>
`;

exports[`Тест Button Тест кнопки с текстом 1`] = `
<button
  className="text text_type_button text_color_primary button undefined false "
  type="button"
>
  <p
    className="text undefined"
  >
    Кнопка
  </p>
</button>
`;
